src/WS.cpp

```diff
@@ -1112,27 +1209,40 @@ signed char WS::stream_callback(struct lejp_ctx *ctx, char reason)
 
     if (reason & LEJP_FLAG_CB_IS_VALUE && ctx->path_match)
     {
-        // LOG_DEBUG("stream_callback: " << u_ctx->path << " = " << (char *)ctx->buf);
+        append_session_msg(
+            u_ctx->message, "%s\"%s\":", (u_ctx->flag & PNT_FLAG_SEPARATOR) ? "," : "", stream_keys[ctx->path_match - 1]);
 
-        append_message(
-            "%s\"%s\":", u_ctx->s ? "," : "", stream_keys[ctx->path_match - 1]);
+        u_ctx->flag |= PNT_FLAG_SEPARATOR;
 
-        if ((ctx->path_match >= PNT_STREAM_GOP && ctx->path_match <= PNT_STREAM_OSD))
+        if (ctx->path_match >= PNT_STREAM_GOP && ctx->path_match <= PNT_STREAM_SCALE_HEIGHT)
         { // integer values
             if (reason == LEJPCB_VAL_NUM_INT)
                 cfg->set<int>(u_ctx->path, atoi(ctx->buf));
-            append_message(
-                "%d", cfg->get<int>(u_ctx->path));
+            append_session_msg(
+                u_ctx->message, "%d", cfg->get<int>(u_ctx->path));
+        }
+        else if(ctx->path_match >= PNT_STREAM_ENABLED && ctx->path_match <= PNT_STREAM_SCALE_ENABLED)
+        { // bool values
+            if (reason == LEJPCB_VAL_TRUE)
+            {
+                cfg->set<bool>(u_ctx->path, true);
+            }
+            else if (reason == LEJPCB_VAL_FALSE)
+            {
+                cfg->set<bool>(u_ctx->path, false);
+            }
+            append_session_msg(
+                u_ctx->message, "%s", cfg->get<bool>(u_ctx->path) ? "true" : "false");
         }
         else
         {
             switch (ctx->path_match)
-            {
+            {                              
```

Comment:
Extra trailing whitespace that needs to trimmed

Reply:
pull incomming.